tests/test_distro.py

```diff
@@ -734,6 +731,34 @@ def test_empty_release(self) -> None:
         desired_outcome = {"id": "empty"}
         self._test_outcome(desired_outcome)
 
+    def test_absolutesymlinks(self) -> None:
+        self.distro = distro.LinuxDistribution(
+            root_dir=os.path.join(TESTDISTROS, "distro", "absolutesymlinks")
+        )
+        desired_outcome = {"id": "absolutesymlinks"}
+        self._test_outcome(desired_outcome)
+
+    def test_rootdirescape(self) -> None:
+        self.distro = distro.LinuxDistribution(
+            root_dir=os.path.join(TESTDISTROS, "distro", "rootdirescape")
+        )
+        # loading existing file outside of root_dir has been prevented (empty data)
+        self._test_outcome({})
+
+    def test_rootdirnonescape(self) -> None:
+        self.distro = distro.LinuxDistribution(
+            root_dir=os.path.join(TESTDISTROS, "distro", "rootdirnonescape")
+        )
+        desired_outcome = {"id": "rootdirnonescape"}
+        self._test_outcome(desired_outcome)
```

Comment:
I understand the idea behind the test itself and consider it a good test, but fail to see `rootdirnonescape` as a good descriptive name. Any other ideas for a name?

Reply:
Definitely, what about `rootdirfollowsymlink` ? 🙂

Reply:
Hi @HorlogeSkynet, I think I was not paying full attention last time and mis-reading "nonescape" as "none escape" rather than "non-escape", which made significantly less sense to me.  Maybe non-escape is not so bad for a name at all, given that the test exercises a case that looks like an escape at first glance and then turns out to be sane, when having a closer look.  Maybe the best fix is to break with the existing naming scheme "test_notveryreadable" (below) at least for new tests and use e.g. "test_root_dir_non_escape" rather than "test_rootdirnonescape" for a name.  I believe readability would beat consistency here.  What do you think?

Reply:
All good for me ! 💯 
Thanks @hartwork 🙏